src/rust/ensogl/example/src/lib.rs

```diff
@@ -4,17 +4,14 @@
 //! the FRP endpoints to communicate with controllers. It also have a mocked debug scenes of IDE.
 
 #![feature(associated_type_defaults)]
-#![feature(clamp)]
 #![feature(drain_filter)]
 #![feature(entry_insert)]
 #![feature(fn_traits)]
-#![feature(overlapping_marker_traits)]
 #![feature(specialization)]
 #![feature(trait_alias)]
-#![feature(type_alias_impl_trait)]
+#![feature(min_type_alias_impl_trait)]
 #![feature(unboxed_closures)]
-#![feature(vec_remove_item)]
-#![feature(weak_into_raw)]
+#![allow(incomplete_features)] // To be removed, see: https://github.com/enso-org/ide/issues/1559
```

Comment:
I dont agree it sohuld be removed. This claim is too hard. I agree we might benefit from investigating further if min specializations will suit our case, but I don't agree that we NEED to remove it. See my comments in the linked issue to learn more.

Reply:
Agreed to left as-is.